source/reports/ml_reports/ROCCurve.py

```diff
@@ -0,0 +1,70 @@
+from source.reports.ReportOutput import ReportOutput
+from source.reports.ReportResult import ReportResult
+from source.reports.ml_reports.MLReport import MLReport
+from sklearn.metrics import roc_curve, auc
+
+import warnings
+import plotly.graph_objs as go
+import os
+import numpy as np
+
+
+class ROCCurve(MLReport):
+
```

Comment:
Could you maybe additionally add documentation (will be then automatically added with the rest)? You can see the Coefficients report, for an example of documentation :) Just a sentence what it is and if there are any params (here there are not) and how YAML specs would look. Specifically here it would be:
```
reports:
  my_roc_report: ROCCurve
```